README.md

```diff
@@ -1 +1,60 @@
-# web-idv-demo-python-flask
+
+# Web IDV Demo in Python using Flask
+
+## Introduction
+
+This project demonstrates the integration of Authenteq Web IDV with the application written in Python.
+It's implemented with Flask and jinja2 for the details page. 
+
+
+## Running project
+python version 3.7 or higher is required
+
```

Comment:
Add requirements.txt file and add here step explaining how to instal dependencies.